src/views/AddExpenses.js

```diff
@@ -20,66 +26,104 @@ const AddExpenses = ({ addExpense, setMonthToShow }) => {
     setTitle('');
     setDate('');
     setAmount('');
+    setWasTitleValidated(false);
+    setWasDateValidated(false);
+    setWasAmountValidated(false);
+  };
+
+  const handleTitleChange = evt => {
+    setTitle(evt.target.value);
+    setWasTitleValidated(true);
+  };
+
+  const handleDateChange = date => {
+    setDate(date);
+    setWasDateValidated(true);
+    setMonthToShow(
+      getUKFormattedDate(date, {
+        year: 'numeric',
+        month: 'long',
+      })
```

Comment:
personal: I find it really hard to read when we wrap a single statement into so many lines. the grouping gets lost.

my analogy is a book. imagine the author of a story would break after every three lines. this would break the whole reading flow.

just my 2 cents...

Reply:
fixed where prettier allows :)